.github/workflows/publish-docker.yml

```
@@ -43,6 +43,7 @@ jobs:
         with:
           context: .
           push: true
+          platforms: linux/amd64,linux/arm64,linux/arm/v7,linux/arm64/v8
```

I saw here https://github.com/docker/build-push-action/blob/master/.github/build-push-summary.png that this also support a list format, which might be easier to understand

 ```suggestion
          platforms: 
            - linux/amd64
            - linux/arm64
            - linux/arm/v7
            - linux/arm64/v8
```